03_probability_intro/03_probability_intro.jl

```diff
@@ -397,6 +397,19 @@ md"
 Considering that we have only tried 100 times, the probabilities have been estimated pretty well! Each distribution assigns a high-enough probability to the true value of the bandit probabilities and it's surroundings. Another strategies rather than the Thompson sampling can be tested to see how well they perform, this was just a simple example to apply Bayesian probability.
 "
 
+# ╔═╡ 49b84f96-7c2f-11eb-3bf8-bde596991317
+md"
+### Summary
+In this chapter we introduced the basic concepts of probability.
+We talked about probability of independent events and also conditional probability to arrive at the Bayes' theorem.
+
+Then we addressed the two main approaches to probability, the frequentist approach and in particular the Bayesian one where our initial beliefs can be updated with the income of new data.
```

Comment:
>  Then, we addressed the two main approaches to probability: the frequentist approach and the Bayesian one, where our initial beliefs can be updated with the addition of new data.